dedalus/libraries/matsolvers.py

```diff
@@ -24,6 +24,9 @@ def __init__(self, matrix, solver=None):
     def solve(self, vector):
         pass
 
+    def solve_adjoint(self,vector):
```

Comment:
Maybe we should all this `solve_H` instead of `solve_adjoint`, to match the interface of superlu and be a little more explicit, in case we/someone wants `solve_T` at some point.

Reply:
Also agreed, have changed this in the code too.